packages/blade/src/components/DatePicker/DatePicker.stories.tsx

```diff
@@ -132,13 +136,25 @@ SingleDatePicker.storyName = 'SingleDatePicker';
 SingleDatePicker.args = {
   label: 'Select a date',
   selectionType: 'single',
+  onInput: (e) => {
```

Comment:
I think what they wanted was, if they do `document.querySelector('input[type="date"]').addEventListener('input', () => {}` then callback should get called (@anuraghazra can you confirm?)

Our onInput and onChange wont dispatch the native DOM events